src/geowombat/__init__.py

```diff
@@ -1,5 +1,5 @@
 __path__: str = __import__('pkgutil').extend_path(__path__, __name__)
-__version__ = '2.1.11'
+__version__ = '2.1.13'
```

Comment:
I'm also confused why this is a change. See [here](https://github.com/jgrss/geowombat/blob/6c8b96bb2c3ed772ff9843a95a87a94ae0a35d5d/src/geowombat/__init__.py#L2).